src/extensions/cms/store.js

```diff
@@ -0,0 +1,59 @@
+import config from 'config'
+import fetch from 'isomorphic-fetch'
+
+const state = {
+  cmsPages: [],
+  cmsBlock: []
+}
+
+const getters = {
+  getBlock: (state) => (id) => {
+    return state.cmsBlock.find(item => item.id === id)
+  },
+  getPage: (state) => (id) => {
+    return state.cmsPages.find(item => item.id === id)
+  }
+}
+
+// actions
+const actions = {
+  loadCms (context, {id, type}) {
+    let url = (config.cms.endpoint)
+      .replace('{{type}}', type)
+      .replace(`{{cmsId}}`, id)
```

Comment:
IMO there is no need to use template strings. You can just use single quotes `'{{cmsId}}'`

Reply:
corrected